src/app/modules/timesheet/timesheet.routes.ts

```diff
@@ -1,14 +1,21 @@
 import { NgModule } from '@angular/core';
 import { Routes, RouterModule } from '@angular/router';
 
+import { HeaderComponent } from 'src/app/components/header/header.component';
+
 import { InvoicePDFComponent } from './components/invoice-pdf/invoice-pdf.component';
+
 import { TimesheetEditComponent } from './components/timesheet-edit/timesheet-edit.component';
 import { TimesheetReviewComponent } from './components/timesheet-review/timesheet-review.component';
 
 const routes: Routes = [
-  { path: 'timesheet/create', component: TimesheetEditComponent },
-  { path: 'timesheet/edit/:data', component: TimesheetEditComponent },
-  { path: 'timesheet/review/:data', component: TimesheetReviewComponent },
+  {
+    path: 'timesheet', component: HeaderComponent, children: [
```

Comment:
Like I said before, It's strange to have on the `parents` route a `HeaderComponent`,  you see what I mean ?